### PR TITLE
Fix remote rendering test pipeline

### PR DESCRIPTION
**Pull request by MichaelZp0**
# Description

The test pipeline for the remoterendering sdk is missing an environment variable, thus all tests run fail in the setup process.
This PR adds this environment variable and should fix the tests or at least will run them, so we can see if something is really broken and not only the environment.

### CODE REVIEW

**Review by ChristopherManthei — APPROVED** [review]

File: sdk/remoterendering/azure-mixedreality-remoterendering/CHANGELOG.md [hunk 1]
```diff
@@ -1,14 +1,8 @@
...
+## 1.0.0b2 (2022-11-03)
 
-### Features Added
-
-### Breaking Changes
-
-### Bugs Fixed
-
-### Other Changes
+- Fixed bug in testing pipeline.
```
Comment (tjprescott) [inline sdk/remoterendering/azure-mixedreality-remoterendering/CHANGELOG.md:5]: This isn't a customer facing change (or even an internal one) and thus nothing should be added to the changelog.

**Review by tjprescott — CHANGES_REQUESTED** [review]

File: sdk/remoterendering/azure-mixedreality-remoterendering/CHANGELOG.md (hunk 1 above)
Comment (MichaelZp0) [reply sdk/remoterendering/azure-mixedreality-remoterendering/CHANGELOG.md]: Okay. I just removed the entry then.

**Review by tjprescott — APPROVED** [review]

**Pull request merged**